src/main/java/org/reaktivity/nukleus/http/internal/routable/stream/SourceInputStreamFactory.java

```diff
@@ -457,20 +459,27 @@ private int decodeHttpBegin(
                 {
                     // Incomplete request, not yet cached
                     slotIndex = slab.acquire(sourceId);
+                }
+                if (slotIndex == NO_SLOT)
```

Comment:
Please add a blank line between the two `if` blocks, otherwise it feels like the `else` is missing.